app/assets/javascripts/admin/enterprises/controllers/enterprise_controller.js.coffee

```diff
@@ -47,7 +50,7 @@ angular.module("admin.enterprises")
           email: manager.email
           confirmed: manager.confirmed
         if (user for user in $scope.Enterprise.users when user.id == manager.id).length == 0
-          $scope.Enterprise.users.push manager
+          $scope.Enterprise.users.splice(0, 0, manager)
```

Comment:
Uh, splice, I just had to look that up again. But isn't `splice(0, 0, x)` the same as `unshift(x)`? That would be neater, wouldn't it?